MapAnalyzer/Pather.py

```diff
@@ -79,20 +102,22 @@ def _add_non_pathables_ground(self, grid: ndarray, include_destructables: bool =
                 if "plates" not in rock.name.lower():
                     self.add_cost(position=rock.position.rounded, radius=1 * rock.radius, arr=grid, weight=np.inf, safe=False)
         return grid
+    
+    def lowest_cost_points_array(self, from_pos: tuple, radius: float, grid: np.ndarray) -> np.ndarray:
+        disk = tuple(draw_circle(from_pos, radius, shape=grid.shape))
 
-    def find_lowest_cost_points(self, from_pos: Point2, radius: float, grid: np.ndarray) -> List[Point2]:
-        # Add 0.01 to radius to find a closed disk
-
-        disk = tuple(skdraw.disk(center=from_pos, radius=radius + 0.01, shape=grid.shape))
         if len(disk[0]) == 0:
```

Comment:
What is the reason that the Exception log was removed in this case ?

Reply:
The function should work fine near map edge when its bounded by grid shape; only reason no points will be returned is if the radius is too small